lib/main.dart

```diff
@@ -51,18 +51,46 @@ class _QuizAreaState extends State<QuizArea> {
 
 
   Widget answerButton({required String answer})
-    => Container(
-      color: Colors.grey[400],
-      child: TextButton(onPressed: (){
-        setState(() {
-          if(questionBank.getQuestionCorrectAnswer() == answer)
-            scoreBoard.add(Icon(Icons.check, color: Colors.green,),);
-          else
-            scoreBoard.add(Icon(Icons.close, color: Colors.red,),);
-          questionBank.nextQuestion();
-        });
-      }, child: Text(answer,
-      textAlign: TextAlign.center,)),
+    => Visibility(
+      child: Container(
+        color: Colors.grey[400],
+        child: TextButton(onPressed: (){
+          setState(() {
+            if(questionBank.getQuestionCorrectAnswer() == answer) {
```

Comment:
you should extract the code in if else brackets to new method with name describing what exactly is going on here

Reply:
agreed, but i wanted to simplify it by doing widget answer array in main dart file, i know it isn't really objective approach, but since this is a widget on the same screen i just left it as it is :P